lineage/states/stateCommon.py

```diff
@@ -50,10 +50,10 @@ def gamma_LL(
     """Log-likelihood for the optionally censored Gamma distribution."""
     x = np.exp(logX)
     glnA = gammaln(x[0])
-    outt = 0.0
+    outt = np.zeros(len(x) - 1)
     for i in range(len(x) - 1):
         gobs = gamma_obs[i] / x[i + 1]
-        outt -= np.dot(
+        outt[i] -= np.dot(
             gammas[i] * time_cen[i],
             (x[0] - 1.0) * np.log(gobs) - gobs - glnA - logX[i + 1],
         )
```

Comment:
sorry it is taking too long. I have been trying to remember how we calculated this likelihood..
Is `outt` the log-likelihood value for each condition (control,conc1, conc2, conc3) ?

Reply:
Yes. I separated the conditions in `outt` because it allows me to do much of the differencing in one call.